templates/cicd/manifest.yml

```diff
@@ -0,0 +1,20 @@
+# This manifest defines the relationship and deployment ordering of your environments.
+
+# This section defines the source artifacts.
+source:
+  # The name of the provider that is used to store the source artifacts.
+  provider: {{.Provider}}
+  # Additional properties that further specifies the exact location
+  # the artifacts should be sourced from.
+  properties:
+    # Repository in the format of <user_name>/<repository_name>.
+    repository: {{.Repository}}
+
+{{$length := len .Stages}}{{if gt $length 0}}
+# The deployment section defines the order the pipeline will deploy
+# to your environments
+deployment: {{range $stage := .Stages}}
```

Comment:
oh lol I was fine with keeping the `.Environments` field. Sorry for the confusion!  
I meant renaming this key from `deployment` to `stages` 😛 since this is public facing